src/App.tsx

```diff
@@ -1,10 +1,54 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import cn from 'classnames';
 import { UserWarning } from './UserWarning';
+import { getTodos } from './api/todos';
+import { Todo } from './types/Todo';
+import { Status } from './types/Status';
+import { getFilteredTodos } from './services/getFilteredTodos';
 
-const USER_ID = 0;
+const USER_ID = 87;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [filter, setFilter] = useState<Status>(Status.All);
+
+  const filteredTodos = getFilteredTodos(filter, todos);
+  const activeTodosCount = todos.reduce((acc, cur) => {
+    if (!cur.completed) {
+      return acc + 1;
+    }
+
+    return acc;
+  }, 0);
+
+  useEffect(() => {
+    let timeout: NodeJS.Timeout;
+
+    getTodos(USER_ID)
+      .then(setTodos)
+      .catch(() => {
+        setErrorMessage('Error');
+        timeout = setTimeout(() => {
+          setErrorMessage('');
+          clearInterval(timeout);
+        }, 3000);
```

Comment:
why do you use clearInterval if you did't use setInterval?

I think we can do it in more simple way, try to use `wait` from `utils/fetchClient.ts`

for example,
```
.catch(() => {
  setErrorMessage('Error');
  return wait(3000).then(() =>  setErrorMessage(''));
});
 ```
 
 but the best option to create a separate component for ErrorMessage, and this setTimeout and clearing of this Timeout should be there